pandora/src/redux/actions/productActions.js

```diff
@@ -66,7 +72,7 @@ export const fetchCategories = () => {
     }
 }
 
-export const fetchProducts = () => {
+/* export const fetchProducts = () => {
```

Comment:
fetchProductsByUserChoices fonksiyonu, default değerleri ile standart fetchProducts değil mi? Diğerinde default value vererek yazdığın için zaten bire bir çalıştırması lazım.